src/components/ProductTableWrapper.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import ProductTable from "./ProductTable";
+import { ProductsCollection } from "../queries/products";
+
+export default class ProductTableWrapper extends React.Component {
+  state = {
+    selectedProduct: null
+  };
+  render() {
+    const { profile, productDeleteConfirm } = this.props;
+    const { selectedProduct } = this.state;
+
+    if (profile.loading) {
+      return <ProductTable isLoading={true} products={[]} />;
+    }
+    const { organization } = profile.data;
+
+    return (
+      <ProductsCollection
```

Comment:
@rubenspgcavalcante Would it make sense to load the data of the query of `ProductsCollection` into the Redux store instead of sending a query each time to the firestore? Hashtag Offline support?